src/components/inspectors/InspectorPanel.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <b-card no-body class="inspector-container border-top-0 border-bottom-0 border-right-0 rounded-0" data-test="inspector-container">
     <vue-form-renderer
+      :key="config[0].name"
```

Comment:
This ensures the form-renderer components are re-rendered when a different config is loaded. This means that at least one default accordion will be opened when switching between different node types.